src/auto/key_file.rs

```diff
@@ -283,7 +293,12 @@ impl KeyFile {
         }
     }
 
-    //pub fn to_data(&self) -> Result<(String, /*Unimplemented*/Fundamental: Size), Error> {
-    //    unsafe { TODO: call ffi::g_key_file_to_data() }
-    //}
+    pub fn to_data(&self) -> Result<(String, usize), Error> {
```

Comment:
This one actually not because it misses the array-length annotation for the return value in GLib. We should go through all these things and report them in Bugzilla...